src/numerics/include/metaphysicl/dualnumber_surrogate.h

```diff
@@ -57,17 +57,33 @@ inline DualNumberSurrogate<T, D>::DualNumberSurrogate(DualNumber<T2, D2> && dn,
 }
 
 template <typename T, typename D>
-inline DualNumberSurrogate<T, D>::DualNumberSurrogate(DualNumberSurrogate<T, D> & dns)
-  : _value(dns.value()), _derivatives(dns.derivatives())
+inline DualNumberSurrogate<T, D>::DualNumberSurrogate(const DualNumberSurrogate<T, D> & dns)
+  : _value(dns._value), _derivatives(dns._derivatives)
 {
 }
 
 template <typename T, typename D>
 inline DualNumberSurrogate<T, D>::DualNumberSurrogate(DualNumberSurrogate<T, D> && dns)
-  : _value(dns.value()), _derivatives(dns.derivatives())
+  : _value(dns._value), _derivatives(dns._derivatives)
```

Comment:
Same question here as above but without the same clarification; I don't have a guess at all...